test/components/cell/Cell.test.js

```diff
@@ -0,0 +1,19 @@
+import Enzyme from 'enzyme';
+import React from 'react';
+import {Cell} from '../../../src/components/cell/Cell';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
I wonder if we can create a helper file to not need to configure the Enzyme adapter each time.